docs/how_to/pytorch_install/pytorch_install.md

```diff
@@ -159,63 +130,62 @@ Follow these steps:
    git submodule update --init --recursive
    ```
 
-4. Build PyTorch for ROCm.
+4. Build PyTorch for ROCm. The default build directory is `rocm/pytorch:latest-base`.
 
    :::{note}
-   By default in the `rocm/pytorch:latest-base`, PyTorch builds for these
-   architectures simultaneously:
-   - gfx900
-   - gfx906
-   - gfx908
-   - gfx90a
-   - gfx1030
-   :::
+   PyTorch builds simultaneously for the following architectures:
```

Comment:
This is not generally true. It's true for the `rocm/pytorch:latest-base` docker image because it has an env variable that specifies these architectures to build for, hence the note.